CLI.md

```diff
@@ -0,0 +1,64 @@
+## Command Line Usage
+
+The CLI tool is provided mainly for quick tests and debugging.
+
+### General help
+
+```shell
+./mindee generated --help
+```
+
+> Note: Due to the limited-nature of most PHP CLI tools, the help sections aren't customized for each command.
+
+### Example parse command for Off-the-Shelf document
+
+```shell
+./mindee invoice -k xxxxxxx /path/to/invoice.pdf
+```
+
+### Works with environment variables
+
+```shell
+export MINDEE_API_KEY=xxxxxx
+./mindee invoice /path/to/invoice.pdf
+```
+
+### Example parse command for a generated document (DocTI)
+
+```shell
+./mindee generated -a pikachu -k xxxxxxx pokemon_card /path/to/card.jpg -A
+```
+
+
+### Example async parse command
+
+```shell
+./mindee invoice-splitter path/to/the/invoice.pdf -A
+```
+
+> Note: the `-A` parameter is **mandatory** for asynchronous APIs.
```

Comment:
If the product only supports async mode, it should not be mandatory to pass this option.
Ideally this option is only used when the product supports both modes.